packages/api/controllers/operatorServices/operations/deletePropertyEnv.js

```diff
@@ -69,23 +89,35 @@ const deletePropertyEnv = async (req, res, next) => {
     await dataAlias.save();
     await activityStream.insertMany(applicationsActivityStream);
 
-    console.log(deleteAlias);
-    console.log(deleteApplication);
+    log.info(deleteAlias);
+    log.info(deleteApplication);
 
```

Comment:
What is the need for logging this two data

Reply:
I've added that in case we need to check the deleted data for the env deletion